hooks/openapi/useRequestParameters.tsx

```diff
@@ -80,10 +81,24 @@ export function useRequestParameters(
 
     const getRequestSchema = (
       schema: OpenAPIV3_1.PathItemObject | null | undefined,
+      method: string,
     ): OpenAPIV3_1.SchemaObject | null => {
-      return (
-        schema?.post?.requestBody?.content?.['application/json']?.schema ?? null
-      );
+      if (method == 'put') {
+        return (
+          schema?.put?.requestBody?.content?.['application/json']?.schema ??
```

Comment:
@suhailkakar I couldn't figure out a way to do something like `schema?[method].requestBody` is there a better way to do this than this big if statement?

Reply:
I believe this should work `schema?.[method]?.requestBody?.content?.['application/json']?.schema ?? null`